src/bramble/client/main.js

```diff
@@ -588,6 +604,27 @@ define([
                 });
             }
 
+            function getChildrenOfFolder(folderPath, callback) {
+                var children = [];
+
+                function storeRelativePath(childPath, next) {
+                    // We don't care about child folders, only files
```

Comment:
Neither Thimble nor publish care about folders (only file paths) and they don't have the ability to detect if a path is a folder or a file.